wdl-engine/src/outputs.rs

```diff
@@ -3,20 +3,21 @@
 use std::cmp::Ordering;
 
 use indexmap::IndexMap;
+use serde::Serialize;
 
 use crate::Scope;
 use crate::Value;
 
 /// Represents outputs of a WDL workflow or task.
-#[derive(Default, Debug, Clone)]
+#[derive(Default, Debug, Clone, Serialize)]
 pub struct Outputs {
     /// The name of the outputs.
     ///
     /// This may be set to the name of the call in a workflow or the task name
     /// for a direct task execution.
     name: Option<String>,
     /// The map of output name to value.
-    values: IndexMap<String, Value>,
+    pub values: IndexMap<String, Value>,
```

Comment:
If we make `Outputs` properly serializable with what's being done in the `serialize` function, I don't think this will need to be public.